packages/swingset-liveslots/src/collectionManager.js

```diff
@@ -348,6 +348,15 @@ export function makeCollectionManager(
       updateEntryCount(1);
     }
 
+    function addToSet(key) {
```

Comment:
```suggestion
    const addToSet = key => {
```

Reply:
Please recheck.

Still grumpy that I can't write `const addToSet = (key) => {` without whining from prettier.  Being consistent about argument list delimiters would address much of my concern about the legibility of this style.

Reply:
Yes, I am sympathetic to this issue. OTOH, there are too many times we write one-arg callbacks essentially as part of a repeated syntactic idiom like `.map(x => something)` or `E.when(p, f => something)`. For all these to pay the readability cost of more punctuation would be worse.

If there's a setting that lets the common idioms stay clean while allowing the others, that may be interesting. But last I looked there isn't.

Reply:
I'd say just allow the parens around a single arg to be optional.  When one is writing an inline function expression that's going be, e.g., passed as a parameter, it makes sense to be terse.  But when writing something that's intended to be a top-level function definition, I find the non-orthogonality to be aggravating.

Reply:
optional means it wont be normalized, and for the inline cases, some will and some won't, making readability even worse than normalizing on the wrong option.

Reply:
I think I just don't weight that normalization nearly as highly as you do.  There are too many cases where the normalized form makes my eyes bleed.  I'll favor legibility over normalization any day of the week in cases where normalization makes the code harder to read (which it sometimes definitely does).  Consistency is good for legibility, and normalization _does_ ensure consistency, but consistency per se is insufficient. We could, for example, have a rule that that says to have one lexical token per line with no indentation, which would be consistent but also insane.  I fundamentally don't think it's a problem to have human aesthetic judgement driving some of how we use whitespace and punctuation for formatting.  We already use human aesthetic judgement with respect to countless other aspects of our code, such as the order that things are placed in a module, the names of functions, methods, and variables, which lines of code need comments vs. which can do without them, and much more, not to mention the design of APIs themselves.  There are even some uses of whitespace that prettier still allows you to use your own judgement on, such as where to insert blank lines to separate one bit of code from an adjacent, less related bit of code (code paragraphs?).  I'm actually pretty obsessive about formatting consistency myself -- my beef with prettier is often not so much that it's opinionated but that some of its opinions are just wrong -- but sometimes I'm following a rule that isn't easily captured by automation (such as, to return to where we started: Inline function expression with a single arg, leaving off the parens is more legible; definition of a top level function with a single arg, adding parens around the arg is more legible -- however, the determination as what falls into which category is subjective and so automation may not be feasible, but I trust the programmer on this, because it's really about expressing their intent anyway).